README.md

```diff
@@ -11,84 +11,96 @@ States, which are stored after a certain interval, are used for that.
+**use_event_publish: _true** — this parameter specifies if the component will publish any events
 
-**use_event_publish: _true** — this parameter specifies if the component will publish any events.
+**disable_cradle_saving: _false_** — this parameter specifies if the component will store the messages to 
+[cradle](https://github.com/th2-net/cradleapi)
 
-**disable_cradle_saving: _false_** — this parameter specifies if the component will store the messages to [cradle](https://github.com/th2-net/cradleapi).
+**disable_connectivity_saving: _false_** — this parameter specifies if the connection event will be stored
 
-**disable_connectivity_saving: _false_** —
+**event_batch_size: _1048576_** — this parameter specifies the size of an event batch (1MB by default)
 
-**event_batch_size: _1048576_** — this parameter specifies the size of an event batch (1MB by default).
+**event_batcher_core_pool_size: _2_** —  this parameter specifies the number of events necessary for sending another 
+event batch
 
-**event_batcher_core_pool_size: _2_** — 
+**event_batcher_max_flush_time: _1000_** — the number of milliseconds for scheduling the sending of event batches
 
-**event_batcher_max_flush_time: _1000_** — 
-
-**use_offset_from_cradle: _true_** — 
+**use_offset_from_cradle: _true_** — this parameter is used for synchronization of PCAP packet sequences with
+packet sequences from Cradle
 
 **individual_read_configurations** — this parameter will be covered in the next section
 
-**required_application_properties_mode: _"OFF"_** — 
-
-**message_batch_size: _1048576_** — this parameter specifies the size of a message batch (1MB by default).
-
-**message_batcher_core_pool_size: _2_** — 
+**message_batcher_core_pool_size: _2_** — this parameter specifies the number of events necessary for sending another
+message batch
 
-**message_batcher_max_flush_time: _1000_** — 
+**message_batcher_max_flush_time: _1000_** — the number of milliseconds for scheduling the sending of message batches
 
-**buffered_reader_chunk_size: _8192_** — 
+**buffered_reader_chunk_size: _8192_** — the buffer size for BufferedInputStream
 
-**check_message_batch_sequence_growth: _false_** — 
+**check_message_batch_sequence_growth: _false_** — this parameter specifies if the component checks whether 
+sequence numbers are incremental in message batches
 
-**check_message_batch_timestamp_growth: _false_** — 
+**check_message_batch_timestamp_growth: _false_** — this parameter specifies if the component checks whether
+timestamps are incremental in message batches 
 
-**tcpdump_snapshot_length: _262144_** — 
+**tcpdump_snapshot_length: _262144_** — this parameter specifies the maximum packet length
 
-**sort_th2_messages: _false_** — 
+**sort_th2_messages: _false_** — this parameter specifies if messages should be sorted by timestamps
 
-**usable_fraction_of_batch_size: _1.0_** — 
+**usable_fraction_of_batch_size: _0.9_** — the fraction of batch size to be used; can be from 0 to 1
```

Comment:
Could you please write the explanation why this parameter is added

Reply:
I don't remember the details, but there were problems occasionally with processing the batches by the Processor if this parameter was set to 1.0.

Reply:
The main problem is a deterrent logic to calculate message size in read and mstore.